sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/schema.yaml

```diff
@@ -6,125 +6,34 @@ fields:
+  description: User who submitted the report
 
 - mode: NULLABLE
   name: created_at
-  type: TIMESTAMP
+  type: STRING
   description: Date decision made
 
 - mode: NULLABLE
-  name: decision_type
-  type: STRING
-  description: The type of decision
-
-- fields:
-  - mode: NULLABLE
-    name: uuid
-    type: STRING
-  - mode: NULLABLE
-    name: name
-    type: STRING
-    description: The name of the policy applied
-  - mode: NULLABLE
-    name: is_illegal
-    type: BOOLEAN
-    description: Signifies if the addon illegal or not
-  - mode: NULLABLE
-    name: parent_uuid
-    type: STRING
-    description: Top level UUID
-  - mode: REPEATED
-    name: enforcement_actions
-    type: STRING
-    description: List of enforcement actions
-  mode: REPEATED
-  name: applied_policies
-  type: RECORD
-  description: Applied policies includes parent_uuid, is_illegal field, name, enforcement_actions, uuid
+  name: entity_id
+  type: INTEGER
+  description: Entity ID
 
-- fields:
-  - mode: NULLABLE
-    name: entity_type
-    type: STRING
-    description: Type of entity
-  - fields:
-    - mode: NULLABLE
-      name: id
-      type: INTEGER
-      description: Attribute ID
-    - mode: NULLABLE
-      name: slug
-      type: STRING
-      description: Slug
-    - mode: NULLABLE
-      name: guid
-      type: STRING
-      description: GU ID
-    - mode: NULLABLE
-      name: name
-      type: STRING
-      description: Name of attribute
-    - mode: NULLABLE
-      name: summary
-      type: STRING
-      description: Summary of the action taken
-    - mode: NULLABLE
-      name: description
-      type: STRING
-      description: Description of the issue
-    - mode: NULLABLE
-      name: release_notes
-      type: STRING
-      description: notes about the release
-    - mode: NULLABLE
-      name: average_daily_users
-      type: INT64
-      description: Average number of daily users
-    - mode: NULLABLE
-      name: last_updated
-      type: TIMESTAMP
-      description: when the job was last updated
-    - mode: NULLABLE
-      name: version
-      type: STRING
-      description: Which version was this addon
-    - mode: NULLABLE
-      name: privacy_policy
-      type: STRING
-      description: Which privacy policy is used
-    - mode: NULLABLE
-      name: created
-      type: TIMESTAMP
-      description: When the attribute was created
-    - mode: NULLABLE
-      name: promoted
-      type: STRING
-      description: Promoted
-    mode: NULLABLE
-    name: attributes
-    type: RECORD
-    description: All the attributes
-  - mode: NULLABLE
-    name: classifier_scores
-    type: STRING
-    description: Top level UUID
-  mode: NULLABLE
+- mode: NULLABLE
   name: entity
-  type: RECORD
+  type: STRING
   description: Entity includes classifier_scores, attributes(RECORD), entity_type
 
 - mode: NULLABLE
-  name: entity_id
-  type: INTEGER
-  description: Entity ID
+  name: uuid
+  type: STRING
+  description: ID of UU
```

Comment:
UUIDs are basically the same thing as GUIDs, a particular form of unique ID.  In this case since there isn't an additional qualifier in the UUID field name I'd presume this is the ID of the decision record.